code/game/objects/items/weapons/RPD.dm

```diff
@@ -407,6 +407,9 @@ var/global/list/RPD_recipes=list(
 		return
 	usr.set_machine(src)
 	src.add_fingerprint(usr)
+	if(get_dist(usr,src)>=1)
```

Comment:
if(!src.Adjacent(usr)), please. Otherwise you can use an RPD through a border window.